### ISSUE TITLE
Sounds

### ISSUE THREAD

**Issue by nsg** [konstructs/client#150]
I'm talking mainly of things like "walking sound", "destroy block sound", ... and so on.

Today the server sends a block list and the block textures from the server. I assume we could send a "sound event list" from the server as well.

There are of course 1000 things more important things to do before this, but, I will open this ticket of someone are interested.

Suggested design:

* The client receives an list of sound-event mapping on connect, in same manner as the textures are provided.
* The client has a set of events and if there are a matching sound mapping the sound is played.

Example event:

* Walk sound
* Walk sound on block of type X
* Destroy block
* Destroy block with tool X
* Press use button on block X

**Comment by Henningstone** [comment]
What about a sound library and some decoder? That would be the thing to think about first I guess, where the sounds should come from :)

**Comment by nsg** [comment]
Yeah, some way to generate the sounds is probably a good idea as well :)
